core/trino-spi/src/main/java/io/trino/spi/expression/StandardFunctions.java

```diff
@@ -36,4 +36,14 @@ private StandardFunctions() {}
     public static final FunctionName IS_DISTINCT_FROM_OPERATOR_FUNCTION_NAME = new FunctionName("$is_distinct_from");
 
     public static final FunctionName LIKE_PATTERN_FUNCTION_NAME = new FunctionName("$like_pattern");
+
+    /**
+     * $in is a function accepting symbol reference and list of expressions
```

Comment:
```
{@code $in(value, array)} returns {@code true} when value is equal to an element of the array,
otherwise returns {@code NULL} when comparing value to an element of the array returns an
indeterminate result, otherwise returns {@code false}
```